recipes/libusb/all/test_package/CMakeLists.txt

```diff
@@ -1,8 +1,12 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.13)
 project(test_package C)
 
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup()
 
+if (APPLE)
+    add_link_options(-framework Security)
```

Comment:
Should the library be responsible for propagating such framework flag to consumers so they don't have to deal with this or is this link option up to the consumers' choice?

Reply:
Could be very tricky since libusb is an automaker and not classical cmake package :-( I'd avoid this if possible

Reply:
I would prefer moving it to recipe side: https://docs.conan.io/en/latest/howtos/link_apple_framework.html#how-to-link-with-apple-frameworks

Reply:
Done! Thanks for the link